bundle/run.sh

```diff
@@ -591,13 +591,14 @@ cat /configs/coap-gateway.yaml | yq e "\
   .apis.coap.address = \"${COAP_GATEWAY_UNSECURE_ADDRESS}\" |
   .apis.coap.externalAddress = \"${FQDN}:${COAP_GATEWAY_UNSECURE_PORT}\" |
   .apis.coap.tls.enabled = false |
-  .apis.coap.authorization.authority = \"https://${OAUTH_ENDPOINT}\" |
-  .apis.coap.authorization.clientID = \"${DEVICE_OAUTH_CLIENT_ID}\" |
-  .apis.coap.authorization.clientSecret = \"${DEVICE_OAUTH_CLIENT_SECRET}\" |
-  .apis.coap.authorization.redirectURL = \"${DEVICE_OAUTH_REDIRECT_URL}\" |
-  .apis.coap.authorization.scopes = [ \"${DEVICE_OAUTH_SCOPES}\" ] |
-  .apis.coap.authorization.audience = \"${DEVICE_OAUTH_AUDIENCE}\" |
-  .apis.coap.authorization.http.tls.useSystemCAPool = true |
+  .apis.coap.authorization.providers[0].name = \"plgd\" | 
```

Comment:
There is an env variable named `DEVICE_PROVIDER` for this.

Reply:
fixed